roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -30,11 +30,24 @@
 
 - name: Configure exports
   template:
-    dest: /etc/exports
+    dest: /etc/exports.d/openshift-ansible.exports
     src: exports.j2
   notify:
   - restart nfs-server
 
+# TODO: Clean up old /etc/exports like this or something
+#- name: register exports
+#  command: cat /etc/exports.d/openshift-ansible.exports
+#  register: exports_out
+#- name: remove exports
+#  line_in_file:
+#    src: /etc/exports
+#    line: {{ item }}
+#    state: absent
+#  with_items: exports_out.stdout.lines
+#  when: exports_out.rc == 0
+#
```

Comment:
I got it figured out, I'll deploy /etc/exports.d/openshift-ansible.exports and remove any entries that exist there from /etc/exports. That way the uninstall playbooks can nuke /etc/exports.d/openshift-ansible.exports without fear of deleting something else even though nothing else will exist in most cases. I'm just running into this because i'm installing, uninstalling, installing...